### PR TITLE
Adding detailed CLI installation instructions to getting-started

### PR DESCRIPTION
**Pull request by nucleardreamer**
This adds more information to the `Getting Started` guide by breaking out the CLI into it's own section. Each OS specific section has links to the latest CLI release tag pulled from github.

image

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: shared/getting-started/cliInstructions/_default.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<div>
+  <ul class="nav nav-tabs" role="tablist">
+    <li role="presentation" class="active"><a href="#cli-osx" aria-controls="cli-osx" role="tab" data-toggle="tab">Mac OSX</a></li>
+    <li role="presentation"><a href="#cli-windows" aria-controls="cli-windows" role="tab" data-toggle="tab">Windows</a></li>
+    <li role="presentation"><a href="#cli-linux" aria-controls="cli-linux" role="tab" data-toggle="tab">Linux</a></li>
```
[the remark this answers is not in the document]
Comment (nucleardreamer) [reply shared/getting-started/cliInstructions/_default.md:5]: This is a very good point! It should be pretty easy to add

File: shared/getting-started/cliInstructions/_default.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          <li>
+            Add that folder (e.g. <code>/home/james/balena-cli</code>) to the PATH environment variable. Check this <a href="https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix" target="_blank">StackOverflow</a> post for instructions. Close and re-open the terminal window so that the changes to PATH can take effect.
+          </li>
+          <li>
+            Check that the installation was successful by running <code>balena version</code>.
+          </li>
+        </ol>
+      </p>
+      <p>
+        For more detailed information, visit the <a href="https://github.com/balena-io/balena-cli/blob/master/INSTALL-LINUX.md" target="_blank">detailed Linux installation instructions</a>.
```
[the remark this answers is not in the document]
Comment (nucleardreamer) [reply shared/getting-started/cliInstructions/_default.md:59]: Agreed!

File: pages/learn/getting-started.md [hunk 3]
```diff
@@ -74,12 +74,13 @@ Alternatively, by clicking on the toggle section of the download button, you als
 <!-- This section is heavily customized based on the device -->
 {{import "getting-started/getDeviceOnDash"}}
 
-## Add release
+## Install the {{ $names.cli.lower }}
+
+Now that we have a device or two connected to a {{ $names.company.lower }} fleet, it's time to install the {{ $names.cli.lower }}. This is the recommended way to deploy code to your devices. Follow the instructions below according to your machine's operating system.
```
Comment (codewithcheese) [inline pages/learn/getting-started.md:79]:
```suggestion
Now that we have a device or two connected to a {{ $names.company.lower }} fleet, it's time to deploy code. The recommended way to deploy code is to use the {{ $names.cli.lower }}. Follow the instructions below according to your machine's operating system.
```

File: pages/learn/getting-started.md [hunk 4]
```diff
@@ -74,12 +74,13 @@ Alternatively, by clicking on the toggle section of the download button, you als
 <!-- This section is heavily customized based on the device -->
 {{import "getting-started/getDeviceOnDash"}}
 
-## Add release
+## Install the {{ $names.cli.lower }}
+
+Now that we have a device or two connected to a {{ $names.company.lower }} fleet, it's time to deploy code. The recommended way to deploy code is to use {{ $names.cli.lower }}. Follow the instructions below to install {{ $names.cli.lower }} to your machine's operating system.
```
Comment (pdcastro) [inline pages/learn/getting-started.md:79]:
```suggestion
Now that we have a device or two connected to a {{ $names.company.lower }} fleet, it's time to deploy some code. We will use the {{ $names.cli.lower }} for this. Follow the instructions below to install it on your computer (development workstation). Choose the tab for your operating system.
```
* As per [style manual](https://github.com/balena-io/docs/blob/v5.30.10/README.md#balena-specific-words), we currently refer to the CLI as "the balena CLI", and `$names.cli.lower` resolves to "balena CLI".
* "time to deploy some code" sounds better than "time to deploy code", no?
* The rest is just my preference. The emphasis on "your computer (development workstation)" is to try and be as clear as possible that they should not install the CLI on their balenaOS device (which some confused users thought they should do in the past), but rather on their computer.

**Review by pdcastro — COMMENTED** [review]
> [...] We wanted the default tab to match the user's OS as reported by the web browser, right? I mean, automatically selected for the user [...]
>> This is a very good point! It should be pretty easy to add

👍   As part of this PR, or in another PR? It could be in another PR, to avoid delaying this one.

File: pages/learn/getting-started.md (hunk 4 above)
Comment (codewithcheese) [reply pages/learn/getting-started.md:79]: In other sentences "your code" is used, should standardize, is some code or your code more accurate

**Comment by nucleardreamer** [comment]
@balena-ci rebase

**Comment by nucleardreamer** [comment]
@balena-ci I self-certify!

**Pull request merged**